homeassistant/components/vacuum/roomba.py

```diff
@@ -5,17 +5,16 @@
 https://home-assistant.io/components/vacuum.roomba/
 """
 import asyncio
+import homeassistant.helpers.config_validation as cv
```

Comment:
It's following PEP8 and alphabetical within that.
https://www.python.org/dev/peps/pep-0008/#imports